src/openapi/streaming/subscription.js

```diff
@@ -331,7 +346,7 @@ Subscription.prototype.reset = function() {
 			return;
 	}
 
-	this.nextAction = null;
+    this.queue.reset();
```

Comment:
Done.

Tried to add jshint indentation rule but they removed it, as author decided that style checking is responsibility of another tool.